### PR TITLE
Add org.gtk.Gtk3theme.Obsidian-2-Aqua

### PR DESCRIPTION
**Pull request by madmaxms**
new Gtk3theme: Obsidian-2-Aqua

### CODE REVIEW

**Comment by madmaxms** [comment]
bot, build org.gtk.Gtk3theme.Obsidian-2-Aqua

File: org.gtk.Gtk3theme.Obsidian-2-Aqua.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{
+   "id":"org.gtk.Gtk3theme.Obsidian-2-Aqua",
+   "branch":"3.22",
+   "runtime":"org.freedesktop.Sdk",
+   "build-extension":true,
+   "sdk":"org.freedesktop.Sdk",
+   "runtime-version":"19.08",
```
Comment (barthalion) [inline org.gtk.Gtk3theme.Obsidian-2-Aqua.json:7]: Can you change it and other variants to 20.08?

**Comment by madmaxms** [comment]
bot, build org.gtk.Gtk3theme.Obsidian-2-Aqua

File: org.gtk.Gtk3theme.Obsidian-2-Aqua.json (hunk 1 above)
Comment (madmaxms) [reply org.gtk.Gtk3theme.Obsidian-2-Aqua.json]: Sure. Done.

**Pull request closed without merging**